python/python/tests/test_transaction.py

```diff
@@ -235,6 +238,7 @@ async def test_transaction_exec_abort(self, redis_client: TRedisClient):
     @pytest.mark.parametrize("cluster_mode", [True])
     @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
     async def test_cluster_transaction(self, redis_client: RedisClusterClient):
+        assert await redis_client.custom_command(["FLUSHALL"]) == OK
```

Comment:
why?

Reply:
because we use the same cluster for resp2 and resp3